core/src/main/scripts/migrate_db.py

```diff
@@ -43,15 +44,21 @@ def __init__(self, database_host, database_name, database_user, database_pw):
         self.database_name = database_name
         self.database_user = database_user
         self.database_pw = database_pw
+        self.database_use_ssl = database_use_ssl
 
 def get_db_cursor(portal_properties):
     """ Establishes a MySQL connection """
     try:
-        connection = MySQLdb.connect(host=portal_properties.database_host,
-            port = portal_properties.database_port,
-            user = portal_properties.database_user,
-            passwd = portal_properties.database_pw,
-            db = portal_properties.database_name)
+        connection_kwargs = {}
+        connection_kwargs['host'] = portal_properties.database_host
+        connection_kwargs['port'] = portal_properties.database_port
+        connection_kwargs['user'] = portal_properties.database_user
+        connection_kwargs['passwd'] = portal_properties.database_pw
+        connection_kwargs['db'] = portal_properties.database_name
+        if portal_properties.database_use_ssl == 'true':
+            connection_kwargs['ssl'] = {"ssl_mode": True}
```

Comment:
could move the `connection_kwargs` creation out of the `try/catch` but not a big deal